requirements-dev.txt

```diff
@@ -171,8 +171,6 @@ rich==10.7.0
     #   molecule
 ruamel.yaml==0.17.13
     # via ansible-lint
-ruamel.yaml.clib==0.2.6
-    # via ruamel.yaml
```

Comment:
@rco-ableton the issue is that `ruamel.yaml==0.17.13` doesn't know about Python 3.10, and only requires `ruamel.yaml.clib` when `python_version<"3.10"` (see https://sourceforge.net/p/ruamel-yaml/code/ci/0.17.13/tree/__init__.py#l16).
This is "fixed" in `ruamel.yaml==0.17.18` (see https://sourceforge.net/p/ruamel-yaml/code/ci/0.17.18/tree/__init__.py#l17), so updating `ruamel.yaml` to `0.17.18` (or its latest, 0.17.20) should make `ruamel.yaml.clib` come back.

Reply:
Nice detective work! I've added https://github.com/Ableton/ansible-role-jenkins-swarm-client/pull/45/commits/fe7d050504f36bd7424546a30ac2d6e4b098e221 and re-generated the dependencies in https://github.com/Ableton/ansible-role-jenkins-swarm-client/pull/45/commits/f2a890bcb679e5e77a1758c2dafa532ef04df3a8. Apologies for re-writing the branch history!